code_editor/views/project_view.py

```diff
@@ -38,43 +30,26 @@ class ProjectView(TemplateView):
 
     # main view to create a project
     def get(self, request, *args, **kwargs):
-        my_form = ProjectForm()
+        form = ProjectForm()
 
-        return render(request, self.template_name, {"form": my_form})
+        return render(request, self.template_name, {"form": form})
 
     # endpoint to create a new project
     def post(self, request, *args, **kwargs):
         project_name = request.POST['project_name']
         description = request.POST['description']
-        language_id = request.POST['language']
-        user = OrmUser.get_user(request.user)
-
-        render_args = {}
-        if project_name in project_names(user):
-            render_args = {
-                "message": f'The project "{project_name}" already exists!', "form": ProjectForm()}
-            return render(request, self.template_name, render_args)
-
-        user_dir = OrmUser.get_user_dir(request.user)
-        language = OrmLanguage.get_language(language_id)
-        language_name = language.language_name
-        language_version = language.language_version
-        project_path = f'media/{user_dir}/{language_name}/{language_version}/{project_name}'
+        language_id = int(request.POST['language'])
+        user = request.user
 
-        # create project in the database and sends the id
-        project = OrmProject.create_project(
-            project_name, description, project_path, language_name, user)
-        project_id = project.id_project
+        project = FileManager.create_project(
+            project_name, description, language_id, user.id)
 
-        # creates file and adds it to the database
-        file = FileManager()
-        file_name = ''
-        if language_name == 'python':
-            file_name = 'main'
-        elif language_name == 'java':
-            file_name = 'Main'
+        if project:
+            return redirect('/api/v1/project/{}'.format(project.id))
```

Comment:
este if es necesario?

Reply:
It checks if project is an object, if it is None it continues and sends a message that tells the user the project already exists.